src/index.js

```diff
@@ -13,5 +13,9 @@ export default [
   'Writing tests takes x3 time than writing actual code.',
   'We need you to work on features. Not performance.',
   'Please do not worry a lot about quality. This is a prototype.',
+  'Since the tests dont pass, just skip them for this delivery build.',
+  'Just make a script to restart the server everyday at 3:00 AM, when nobody uses it, to get around memory leaks.',
+  'You can fix issues as they rise in order to be ready to ship the day after the FAT finishes.',
```

Comment:
Question @giannissmirnios: what does "FAT" mean? Please excuse my ignorance.

Reply:
FAT = Factory Acceptance Test, a usually manual testing before sending the delivery to the client/user wherever it needs to be delivered

Reply:
Not sure if everyone will get this. @Return-1 what say you?

Reply:
Agreed. I didn't know what FAT is either

Reply:
Then maybe remove this line all together - although niche jokes are the best :) Even if not everyone understands it, those who do will find it extra funny.

Reply:
Then maybe remove this line all together - although niche jokes are the best :) Even if not everyone understands it, those who do will find it extra funny.